app/DynamicsAdapter/DynamicsAdapter.Web/Startup.cs

```diff
@@ -68,6 +70,17 @@ private AppSettings ConfigureAppSettings(IServiceCollection services)
      
         }
 
+        private void ConfigureAutoMapper(IServiceCollection services)
+        {
+            var configuration = new MapperConfiguration(cfg =>
+            {
+                cfg.AddProfile<SearchRequestAutoMapperProfile>();                
+            });
+            // only during development, validate your mappings; remove it before release
```

Comment:
why this comment?

Reply:
this is from automapper code, shall we just remove comment or remove comment and valiation?